airbyte-webapp/src/views/Connector/ServiceForm/useAuthentication.tsx

```diff
@@ -202,9 +207,22 @@ export const useAuthentication = (): AuthenticationHook => {
     [advancedAuth, isAuthButtonVisible, legacyOauthSpec]
   );
 
+  const hasAuthFieldValues: boolean = useMemo(() => {
+    if (
+      implicitAuthFieldPaths.some((path) => {
+        return getIn(values, path) !== undefined;
+      })
+    ) {
+      return true;
+    }
+
+    return false;
```

Comment:
Minor code nitpick, since we're anyway just returning `true` and `false` here:

```suggestion
    return implicitAuthFieldPaths.some((path) => getIn(values, path) !== undefined);
```